roles/load-ycsb/templates/workload.yml.j2

```diff
@@ -0,0 +1,53 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: 'ycsb-data-load-job-{{ ycsb_workload }}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+  spec:
+    template:
+      metadata:
+        name: '{{ meta.name }}-ycsb-data-load'
+        namespace: '{{ operator_namespace }}'
+        labels:
+          name: 'ycsb-load-{{ trunc_uuid }}'
+      spec:
+        containers:
+        - name: ycsb
+          image: "quay.io/cloud-bulldozer/ycsb-server:latest"
+          imagePullPolicy: Always
+          env:
+          - name: clustername
+            value: {{ clustername }}
+          - name: es
+            value: {{ elasticsearch.server | default()}}
+          - name: es_port
+            value: {{ elasticsearch.port | default()}}
+          - name: es_index
+            value: {{ es_index }}
+          - name: user
+            value: {{test_user | default()}}
+          - name: workload
+            value: {{ycsb_workload}}
+          - name: num_records
+            value: {{ycsb.recordcount}}
+          - name: num_operations
+            value: {{ycsb.operationcount}}
+          - name: uuid
+            value: {{uuid}}
+          imagePullPolicy: Always
+          command: ["/bin/sh", "-c"]
+          args: ["python /opt/snafu/run_snafu.py --tool ycsb -r 1 -l -d {{ ycsb.driver }} -u {{ uuid }} --user {{ test_user | default("ripsaw") }} -w {{ ycsb_workload }} -x '{{ ycsb.options_load }}'"]
```

Comment:
According to CI, looks like there's a problem with the quotes
```
yaml.scanner.ScannerError: while scanning a plain scalar\\n  in \\\"<string>\\\", line 40, column 168:\\n     ... w workloada -x '-p mongodb.url=\\\"mongodb://mongo-0.mongo/ycsb?\\\"'\\\"]\\n                                         ^\\nfound unexpected ':'\\n  in \\\"<string>\\\", line 40, column 175:\\n     ... oada -x '-p mongodb.url=\\\"mongodb://mongo-0.mongo/ycsb?\\\"'\
```

Reply:
yep this was the error i was talking about on the last wednesday call, i'm not quite sure what would work here..